### DISCUSSION TITLE
More than one 'output' from a Businessrule

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#3382] [Questions] [answered]
Do we see any problem with calling context.AddOutValue twice in Execute method of a business rule?
Something like

```cs
 context.AddOutValue(PrimaryProperty, tax);
context.AddOutValue(FeeProperty, fee);
```

**Comment by rockfordlhotka** [comment, marked as answer]
This is the idea behind affected properties. In the rule's constructor it is necessary to list the properties affected by the rule.

**Reply by Chicagoan2016** [reply]
@rockfordlhotka Thank you
